e2e/runner_test.go

```diff
@@ -184,3 +192,39 @@ func parseFeatureFlags(flags []string) ([]string, []*contextMetadata) {
 
 	return featurePaths, metadatas
 }
+
+// startRuntimeDependencies spins up the runtime dependencies for a stack, represented
+// by a docker-compose file, It will panic if they cannot be satisfied
+func startRuntimeDependencies(stackName string, env map[string]string, minutesToBeHealthy time.Duration) {
```

Comment:
Should we keep `stackName` here? Looks like it has been removed from other similar functions?

Reply:
I think this cannot be removed, as we need to start the proper runtime dependencies for the test suite, in this case the stack-monitoring tests. As this method belongs to the test runner, it's an abstraction to be reused with different test suites (metricbeat, stack-mon...), so it needs to know which stack to run